assets/external_plugins_instructions/rizzler.md

```diff
@@ -0,0 +1,31 @@
+# Rizzler
+
+## Description
+Rizzler rates your conversational Rizz and tells you how to improve your rizz to be a better Rizzler. Rizzle away with the Rizzler.
+
+## Setup Instructions
+1. No additional setup is required for this plugin. It works out of the box with your FRIEND app!
+
+## Usage Instructions
+1. Enable the Rizzler plugin in your FRIEND app settings.
+2. Start a conversation or join a voice chat within the FRIEND app.
+3. After your conversation, the Rizz Coach will automatically analyze the transcript.
+4. You'll receive a fun report including:
+   - A hilarious "Rizz Score" from 0 to 100
+   - A silly "Rizz Title" based on your score
+   - Three humorous tips for improving your conversation skills
+   - A bonus tip for extra laughs
+
+## Example Output
+```json
+{
```

Comment:
Hey, the app wouldn't be able to do anything with this output, the only field you can use is the "message"which will trigger a notification and also be sent to the chat.